src/client/mod.rs

```diff
@@ -1301,12 +1301,15 @@ impl IotHubClient {
         status_code: std::os::raw::c_int,
         context: *mut ::std::os::raw::c_void,
     ) {
-        trace!("SendReportedTwin result: {status_code}");
-
+        let succeeded = status_code == 204;
```

Comment:
missing adapt of version in Cargo.toml

Reply:
no, there is no update since the previous build failed